test/dataCollectionAppTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+require_once ('../getFunctions.php');
+
+use PHPUnit\Framework\TestCase;
+
+class dataCollectionAppTest extends TestCase {
+
+    public function testgetNameImg() {
+        $expected = "<img alt=\"Circle\"src=\"Url\" />";
+        $actual = getNameImg(["ShapeName" => "Circle", "imgurl" => "Url"]);
+
+        $this->assertSame($expected, $actual);
+    }
+
+    public function testErrorWhenString() {
+        $this->expectException(TypeError::class);
+
+        getNameImg("For the Error!");
+    }
+
+    public function testErrorWhenEmpty() {
+        $this->expectException(ArgumentCountError::class);
+
+        getNameImg();
+    }
+
+    public function testgetShapeName() {
+        $expected = "Circle";
+        $actual = getShapeName(["ShapeName" => "Circle"]);
+
+        $this->assertSame($expected, $actual);
+    }
+
+    public function testErrorWhenString2() {
```

Comment:
if you name your tests as mentioned above, you wont need to number your tests, which makes them much easier to understand. although you dont actually need this test as you dont need the function